modules/ROOT/pages/3.6_Data_Structures.adoc

```diff
@@ -18,7 +18,7 @@ The columns of the tables are to be interpreted as follows:
 * Column 21: *Cardinality*, indicates whether the element is mandatory or optional and its multiplicity
 * Column 22: *PropertyDataType*, the type of data expected according to the types defined in the code list ResponseDataType
 * Column 23: *ElementUUID*, The Universal Unique Identifier assigned to identify unambiguously a criterion, group or subgroup. These UUID are kept in e-Certis, except for those that have to be generated dynamically (e.g. UUIDs for Questions, Captions and Requirements). See also the special note "Criteria Taxonomy/Data Structures and the use of UUIDS" below
-* Column 24: *Element Code*; in the case of Criterion it contains the code that categorises the criterion in the taxonomy of exclusion and selection criteria (it maps to the taxonomy drawn in the online documentation and in the spread-sheet files "ESPD-CriteriaTaxonomy*" in folder dist/cl). For the rest of elements there are three types of codes (see also section:
+* Column 24: *Element Code*; in the case of Criterion it contains the code that categorises the criterion in the taxonomy of exclusion and selection criteria (it maps to the taxonomy drawn in the online documentation and in the spread-sheet files "ESPD-criterion*" in folder /codelists/). For the rest of elements there are three types of codes (see also section:
```

Comment:
The sentence seems incomplete
'For the rest of elements there are three types of codes (see also section:'

Reply:
Removed the incomplete sentence.